man/man8/zfs-load-key.8

```diff
@@ -166,6 +166,29 @@ will become one. Alternatively, the
 .Fl i
 flag may be provided to cause an encryption root to inherit the parent's key
 instead.
+.Pp
+Changing the user's key does not change the underlying master key used to
+encrypt the dataset.  Accordingly, if the master key has been compromised,
```

Comment:
How about adding a sentence here: `... underlying master key used to encrypt the dataset.  Therefore, if a user key is compromised, change-key does not necessarily protect newly-written data from attack.`  I think this is the key (haha) warning that we need to get across.  The rest of the new text is describing why this is the case (explaining how the master key wrapping works), and potential mitigations (run `zpool initialize`)